action_server/src/robocorp/action_server/cli.py

```diff
@@ -333,7 +333,7 @@ def _create_parser():
     return base_parser
 
 
-def main(args: Optional[list[str]] = None, *, exit=True) -> int:
+def main(args: Optional[List[str]] = None, *, exit=True) -> int:
```

Comment:
The action server itself is only Python 3.10 onwards (as it's not a library used by clients and is meant to be used as an executable we don't need to use older patterns -- meaning `list[str]` is preferred over `List[str]`).

Reply:
OK. Will revert.